Meadowlark-js/backends/meadowlark-opensearch-backend/package.json

```diff
@@ -27,6 +27,7 @@
   "devDependencies": {
     "@short.io/opensearch-mock": "^0.3.1",
     "copyfiles": "^2.4.1",
+    "dotenv": "^16.0.3",
```

Comment:
The dotenv dependency could be in the root now given that all packages are using it, currently, we have 5 dependencies of dotenv and we've seen in the past that this could cause difference on versions through packages